core/client/src/main/java/alluxio/client/block/stream/NettyPacketWriter.java

```diff
@@ -157,7 +166,7 @@ public void writePacket(final ByteBuf buf) throws IOException {
         try {
           if (!mBufferNotFullOrFailed.await(WRITE_TIMEOUT_MS, TimeUnit.MILLISECONDS)) {
             throw new IOException(
-                String.format("Timeout to write packet to %d @ %s.", mId, mAddress));
+                String.format("Timeout to write packet %s to %s.", mPartialRequest, mAddress));
```

Comment:
Nit: probably do:
Timeout to write packet to %s (address) for request: %s (request). because the protobuf text will spread on multiple lines.